A55-xds-stateful-session-affinity.md

```diff
@@ -455,6 +455,24 @@ following changes are required:
 Note that if stateful session affinity and retries (gRFC [A44][]) are both
 configured, all retry attempts will be routed to the same endpoint.
 
+### Temporary environment variable protection
+
+During initial development, this feature will be enabled by the
+`GRPC_EXPERIMENTAL_XDS_ENABLE_HOST_OVERRIDE` environment variable. This
+environment variable protection will be removed once the feature has
+proven stable. The environment variable protection will specifically apply
+to the following:
```

Comment:
It should also apply to supporting the HTTP filter.

Reply:
Lines 468-469 say that but I can improve the wording if you feel